### DISCUSSION TITLE
Should we kill the google group?

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#997] [General]
Should we kill the google groups? -- Both of them for users and dev? 
Or continue with both this and the google groups?

**Comment by tpike3** [comment]
Personally I struggle keeping up with all the venues, so I would vote to discontinue google groups and just use gihthub discussions

**Comment by Corvince** [comment]
I don't think we have to actively kill the mailing list, but rather let it slowly die. We should definitely post an announcement though and also update the links in the docs. Discussions are a relatively new feature of GitHub and I am not sure lot of people will find it otherwise. We should also probably provide a link in the issue template, so people trying to open an issue can think about whether it makes sense to rather open a discussion.

Re dev list: I would rather move it here as well. I don't remember ever having any discussions that should have been closed to the dev team, I think normally everything can be discussed in the open. And in the rare cases where things should be discussed more private we can just use normal emails.

**Comment by jackiekazil** [comment]
Great! We are in agreement. I will post something tomorrow, then update here on the changes I made.
